airflow-core/src/airflow/ui/public/i18n/README.md

```diff
@@ -350,13 +361,27 @@ FAIL_WHEN_ENGLISH_TRANSLATION_CHANGED = True
 ```
 
 This fails any attempt to change English translation files in a PR unless `allow translation change`
-label is applied to the PR. This allows to still fix critical issues in English translation files and make
-deliberate updates to it but avoids accidental changes.
+label is applied to the PR. This still allows issues in the English translation files to be fixed and
+deliberate updates to be made, while avoiding accidental changes.
 
-Any change in such English translation files during freeze time MUST be communicated in the
-`#18n` Slack channel - so that translators can be informed as early as possible about those translations
+Any change in the English translation files during freeze time MUST be communicated in the
+`#18n` Slack channel and MUST be approved by at least 1 PMC member - so that translators can be informed as early as possible about those translations
```

Comment:
```suggestion
[#18n](https://app.slack.com/client/TCQ18L22Z/C09D0A7FESJ?) Slack channel and MUST be approved by at least 1 PMC member - so that translators can be informed as early as possible about those translations
```